buildpack-deps/Dockerfile.fedora.tpl

```diff
@@ -0,0 +1,26 @@
+FROM #{FROM}
+
+RUN dnf install -y \
+		autoconf \
+		make automake gcc gcc-c++ kernel-devel \
```

Comment:
I see that there is a package named `@development-tools`, but in any case this package differs from distro to distro and doesn't necessarily contain the same build packages across all of them. Let's keep a consistent coding style